### DISCUSSION TITLE
BT-142 winner selection status codes

### DISCUSSION THREAD

**Discussion by EEanalyst** [OP-TED/eForms-SDK#345] [Q&A] [answered]
If the procurement is divided into lots and we are now creating a result notice, do we have to add winner-selection-status code for each lot separately? And if for one lot the winner isn't yet chosen, should we use 'open-nw' code?

**Comment by YvesJo** [comment, marked as answer]
Indeed status of the Lot award has to be specified per lot, or more precisely per LotResult.
Yes, "open-nw" is foreseen for lots not yet awarded because their procedure is still ongoing.
Please note that this question is not an SDK related technical question and should normally be sent to the helpdesk: ted@publications.europa.eu
KR
